narps_open/pipelines/matlabbatch_UK24.m

```diff
@@ -0,0 +1,292 @@
+% (realignment) parameters.
+% The method was adapted from Power et al. (2012), 
+% using 50mm as the assumed radius for translating rotation parameters 
+% into displacements, assuming small angle approximations. A threshold of 
+% 0.5 mm was set to define so-called 'motion outlier' volumes, which after 
+% application to the FD timeseries yielded a binary 'scrubbing' regressor 
+% per run.
+
+% Note: here we can use FramewiseDisplacement from nipype that to compute
+% the framewise displacement from the motion parameters that are available
+% in the rp_ files generated after realignment.
+
+
+% 7 - Generation of nuisance regressors for 
+% 1st level analysis, including motion scrubbing regressor defined using 
+% framewise displacement, and two tissue signal regressors (white matter 
+% and cerebrospinal fluid) derived from averaging the voxel values found 
+% within the segmentaed tissue maps.
+
+% Scrubbing nuisance regressor: any functional volume that has a
+% framewise displacement greater than 0.5mm is set to one in the scrubbing 
+% nuisance covariate (all other volumes are set to zero)
+
+% --> Save as reg_scrubbing.txt
+
+% white matter signal nuisance: average value of voxels in th functional
+% volume that are part of white matter (as computed using the segmentation,
+% threshold>0.5?)
+
+% --> Save as reg_wm.txt
+
+% CSF signal nuisance: average value of voxels in th functional
+% volume that are part of white matter (as computed using the segmentation,
+% threshold>0.5?)
+
+% --> Save as reg_csf.txt
+
+% "independent_vars_first_level": "The GLM design matrix for the 1st level 
+% analysis consited of 17 explicitly specified regressors per run. Of these
+% regressors, the first 8 related to experimental conditions, whereas the 
+% remaining 9 were nuisance regressors (6 head motion parameter regressors,
+% 2 tissue signal regressors, and 1 scrubbing regressor), none of which 
+% were entered as interactions. 
+% For the experimental conditions, we are 
+% interested in the parametric effect of gain and of loss as related to a 
+% mixed gambles task. Trial data, indicating the amount of possible gain 
+% and the amount of possible loss, as well as trial timing, trial durations
+% and reaction times were supplied for each run per subject. From this data
+% a block-task regressor was derived respectively for all trials where the 
+% gamble could result in a possible gain, in a possible loss, and in a 
+% no-loss-nor-gain situation (i.e. 3 block-task regressors). In each case 
+% of the possible gain/loss regressors, 2 parametric modulation regressors 
+% were added: 1 to model the parametric size of the possible gain/loss, and
+% 1 to model the parametric size of the reaction time. In the case of the 
+% no-loss-nor-gain block-task regressor a single extra parametric 
+% modulation regressor was added to model the parametric size of the 
+% reaction time. 
+% All parametric modulation regressors were mean centered 
```

Comment:
Is it the mean value computed for each run, or one for all runs ?

Reply:
@bclenet 'mean centering' should be done automatically as part of the design (this is an option in the model). I can't open Matlab (no more licenses) but I'll check as soon as I can :)